tests/std/tests/P2517R1_apply_conditional_noexcept/test.compile.pass.cpp

```diff
@@ -0,0 +1,19 @@
+#include <string>

```

Comment:
Fun fact: the internal test harness doesn't run tests that aren't added to `tests/std/test.lst`.

Reply:
This file is also missing our normal copyright/license banner.